README.md

```diff
@@ -34,6 +36,21 @@ These status codes and messages appear in transactions created by the DEX (as pa
 - `t1,${takerChain},${takerOrderId},${makerOrderCount}: Orders taken`
 - `t2,${makerChain},${makerOrderId},${takerOrderId}: Order made`
 
+- Taker is the account/wallet who takes the trade from someone else (the maker).
+- Maker is responsible for initialising the trade.
+- All Market orders makes up takers.
+- Pending limit orders in orderbook makes up makers.
+- Limit orders can be makers or takers or both.
+  1. If limit order is fullfilled immediately as soon as placed, it is a taker.
+  2. If limit order goes into pending state after placed, it is a maker.
+  3. If limit order is partially filled after placed (It's a taker), then it goes into pending state (It's a maker).
```

Comment:
`(it's a taker)` and `(it's a maker)` - Lower case since it's not a new sentence.